wagtailmakeup/management/commands/make_up.py

```diff
@@ -84,8 +81,8 @@ def handle(self, *args, **options):
             self.save_image(photo)
 
         # Update all images
-        for image in Image.objects.all():
-            self.update_wagtail_image(image, photos)
+        for photo, images in zip(photos, self.chunk_images(len(photos))):
+            images.update(file=f"unsplash-{photo.id}")
```

Comment:
Won't image be a list var here, returned from `chunk_images`? `.update()` won't work.

Reply:
Slicing querysets generally produces querysets, but i'm not sure how that fairs with the mutation that `chunk_images` is doing.

In my testing it did return lists, although I thought that was something going weird as I had no images locally. If it doesn't work I'll just have to tweak `chunk_images`, but that shouldn't be too difficult.